### PR TITLE
2048 Alphabet Edition

### PR DESCRIPTION
**Pull request by OptimalBlock489**
This is an alphabet edition of the original game 2048 by Gabrielle Cirulli. Here you need to join common alphabets to get a higher alphabet. If you reach, "L", you win. I am personally a huge fan of 2048 so I created this modified version of it for sprig. I hope you like it !

### CODE REVIEW

**Comment by Omay238** [comment]
There are two games like this, and I feel like it would be worthy of submission if you added some other feature, maybe make it longer/harder, like maybe making "L" the last letter.

**Comment by OptimalBlock489** [comment]
Will do. I will make "L" the last letter.

**Comment by OptimalBlock489** [comment]
I made "L" the last letter. I don't think increasing more will be good idea because it is already really hard to reach that level. I can't even reach "F" haha.

**Comment by Omay238** [comment]
I can't get far either. Some people are really good. If you used code from [this game](https://editor.sprig.hackclub.com/?file=https://raw.githubusercontent.com/hackclub/sprig/main/games/2048.js) please add credit to linkai101.

**Comment by OptimalBlock489** [comment]
hmm, I did modify his/her code. I have given the credits.

**Comment by LucasHT22** [comment]
> hmm, I did modify his/her code. I have given the credits.

credits must be at the top of the code

**Review by LucasHT22 — COMMENTED** [review]
please add an image that is from your own game

**Comment by OptimalBlock489** [comment]
I have done the necessary changes. Thank you.

File: games/2048 Alphabet Edition.js [hunk 1]
```diff
@@ -0,0 +1,530 @@
...
+
+Why play 2048 when it's the same basic game you've been playing for years?
+Why don't you move on to a better version? A more intuitive version?
+I present to you... 2048 Alphabet Edition!
+
+Controls:
+Use the a, w, d, and s keys to move left, up, right, and down!
+Instructions:
+2048 is a game where you combine the sums of numbered tiles in order to gain a higher numbered tile.
+This is a spin-off of that game. Try your best to get to "k" and win the game. Good Luck!
```
Comment (LucasHT22) [inline games/2048 Alphabet Edition.js:15]: k?

Comment (OptimalBlock489) [reply games/2048 Alphabet Edition.js:15]: Oops forgot to edit.

Comment (LucasHT22) [reply games/2048 Alphabet Edition.js]: ok!

**Review by LucasHT22 — APPROVED** [review]
Congrats!
If you're a teen, you can get your sprig here! https://sprig-order.hackclub.dev/

**Comment by OptimalBlock489** [comment]
Thanks a lot !!

**Pull request merged**